frontend/src/BookOverview.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useEffect, useState} from 'react';
+import {BookModel} from "./BookModel";
+
+import axios from "axios";
+
+
+function BookOverview() {
+    const [books, setBooks] = useState<BookModel[]>([]);
+
+
+    const fetchAllBooks = () => {
+        axios.get("/api/books")
+            .then(response => response.data)
+            .catch(error => console.error("GET Error: " + error))
+            .then(setBooks)
+    }
+
+
+    useEffect(fetchAllBooks,[])
+
+
+    return <>
+            <p>Book List:</p>
```

Comment:
should the \<p> tag be within the section?

Reply:
done: move p html tag to section